api/openapi.yml

```diff
@@ -77,6 +77,21 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/AuthorizationData'
+  /characters/create:
```

Comment:
if u want to avoid providing user id for urls that works only for logged user, we can agree for a standard for logged user, e.g. `current`
e.g. `/users/current/characters`